core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -323,37 +328,36 @@ __wasi_errno_t
 wasmtime_ssp_clock_res_get(__wasi_clockid_t clock_id,
                            __wasi_timestamp_t *resolution)
 {
-#ifdef BH_PLATFORM_WINDOWS
-    return __WASI_ENOSYS;
+
+    #ifdef BH_PLATFORM_WINDOWS
+    os_win_clock_time_get(clock_id, resolution);
```

Comment:
we should probably name the function `os_clock_time_get` so we don't need those ifdefs here.